games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
+  "category": "GAME",
+  "description": "The well known game. Destroy the invaders with your ship.",
+  "images": [
+    "./Space Invaders [David Winter] - main screen.png",
+    "./Space Invaders [David Winter] - game play.png"
+  ],
+  "interpreter": {
+    "id": "chip8-david-winter",
+    "name": "CHIP8 by David Winter"
+  },
+  "binaries": {
+    "names": [
+      { "value": "Space Invaders [David Winter] - 0.9.1" },
+      { "value": "Space Invaders [David Winter]" },
+
+      { "value": "Space Invaders [David Winter] - 0.9.0" },
+      { "value": "Space Invaders [David Winter] (alt)" },
+      { "value": "INVADERS" }
+    ],
+    "extensions": [
+      "c8",
+      "ch8",
+      ""
+    ],
+    "hashes": [
+      {
+        "type": "SHA256",
+        "value": "00bb7001de52b562d98357d33375406c6d0308466d7d95c2580537d2451a75cd"
+      },
+      {
+        "type": "SHA256",
+        "value": "2d0e1fa53216b297e74041d4fb766f42327a42893e83bb4ec931a9dff5c2dd10"
+      }
+    ]
+  },
+  "sources": {
+    "names": [
+      {
+        "type": "DISASSEMBLED",
+        "value": "Space Invaders [David Winter] - 0.9.0"
+      },
+      {
+        "type": "DISASSEMBLED",
+        "value": "Space Invaders [David Winter] - 0.9.1"
+      }
+    ],
+    "extensions": [
+      "8o",
+      "asm",
+      "txt",
+      ""
+    ]
+  },
+  "platform": {
+    "name": "chip8",
+    "startAddress": "0x200",
+    "resolution": {
+      "width": 64,
+      "height": 32
+    }
+  },
+  "keys": [
+    {
+      "label": "Start",
+      "action": "START",
+      "stroke": "ENTER",
+      "value": "0x5"
+    },
+    {
+      "label": "Fly left",
+      "action": "LEFT",
+      "stroke": "LEFT",
+      "value": "0x4"
+    },
+    {
+      "label": "Fly right",
+      "action": "RIGHT",
+      "stroke": "RIGHT",
+      "value": "0x6"
+    },
+    {
+      "label": "Shoot",
+      "action": "X",
+      "stroke": "SPACE",
+      "value": "0x5"
+    }
+  ],
+  "options": {
+    "opsPerSec": 1500,
+
+    "shiftOp": "MODERN",
+    "loadStoreOp": "UNUSED",
+    "yClipping": "UNUSED",
+
+    "stageColor": "0x808080",
+    "backgroundColor": "0x000000",
+    "foregroundColor": "0xFFFFFF"
+  }
```

Comment:
`stageColor` - would mean the color behind everything (for cases when the area of the app doesn't fully cover the window / screen)
`backgroundColor` - "OFF" pixels
`foregroundColor` - "ON" pixels

Not sure about other colors (xo specific, buzzer specific). Some apps would benefit from textured pixels (something similar to "Brick Game" theme) or post rendering effects (like the CRT and glow in your Space Invaders emulator)

Reply:
Interesting ideas. I think it's best to keep it simple for now though, and hold off on shader effects. A stage color is a good idea for some added flavor.